### PR TITLE
Add Hubspot for stats

### LINKED ISSUE #2471: Send hubspot event when user visits new dashboard
http://developers.hubspot.com/docs/methods/contacts/create_or_update

### PR DESCRIPTION
**Pull request by viddo**
Fixes #2471 (and piggyback for #2572 since so small)

Only load and track Hubspot stats if set in app config.

### CODE REVIEW

**Comment by viddo** [comment]
retest this please

**Comment by viddo** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2032/))

**Comment by viddo** [comment]
@xavijam small change, but please review just in case

File: app/controllers/admin/visualizations_controller.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@ def index
     layout = new_dashboard ? 'new_application' : 'application'
 
     respond_to do |format|
-      format.html { render view, layout: 'application' }
+      format.html { render view, layout: layout }
```
Comment (xavijam) [inline app/controllers/admin/visualizations_controller.rb]: Thanks :+1:

**Comment by xavijam** [comment]
Update NEWS and ready to go :+1:

**Comment by viddo** [comment]
Pending config changes for staging/production to be able to merge, so not ready quite yet.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2038/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2040/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2042/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2056/))

**Comment by viddo** [comment]
Config updates done and verified working on staging, will release as soon as CI pass.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2065/))

**Pull request merged**